src/main/java/edu/umich/its/spe/SimpleJavaEmail.java

```diff
@@ -0,0 +1,82 @@
+package edu.umich.its.spe;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/*
+ * Create generator of java mail sender objects.  Use this to deal with custom javamail property handling.
+ */
+
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Properties;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+
+public class SimpleJavaEmail {
+
+	static final Logger M_log = LoggerFactory.getLogger(SimpleJavaEmail.class);
+
+	Properties mailProperties;
+
+    // Convert hashmap into properties after selecting the right ones.
+
+	// Specify prefix for the mail properties
+	SimpleJavaEmail(HashMap<String,String> inputProperties) {
+		this(inputProperties,"mail.");
+	}
+
+	SimpleJavaEmail(HashMap<String,String> inputProperties,String prefix) {
```

Comment:
I feel `inputProperties` can have a better name and also `prefix` what is it prefixing like emailPrefix.  Both names are too genric. Same thing on line 30

Reply:
Will do.